src/components/Flow/EmployeeSelfOnboardingFlow/EmployeeSelfOnboardingFlow.test.tsx

```diff
@@ -71,7 +71,7 @@ describe('EmployeeSelfOnboardingFlow', () => {
       )
     })
 
-    it('succeeds', async () => {
+    it.skip('succeeds', async () => {
```

Comment:
Was this intentional? If so, should we add a comment explaining why it's skipped?

Reply:
Fixing this now